### DISCUSSION TITLE
Can a Shared Element Transition take place between different components, or should it always be screens?

### DISCUSSION THREAD

**Discussion by Wolk9** [software-mansion/react-native-reanimated#6846] [Q&A]
### 3 accordions
I have three accordions that starts with two unfolded and the third folded. 

When the right action takes place in accordion 2, a descibing component (ItemView) should move to the opening third accordion whilst the first closes. 

If I read what the Shared Element Transition is taking care off it is exactly that, but I only see examples for screens and if I implement it, I get 'duplicate tag' warnings as both source and target View locations are considered on the same screen. 

1) Am I right? 
2) If not, how do I fix it? 
3) If so, I would love to see some tips and ideas
4) If so, wouldn't it be great if this would work inside one 'screen' as well?

**Comment by MatiPl01** [comment]
As far as I know, Shared Element Transitions work only while animating between different screens in the stack navigator (you have to use the native-stack). In other cases, you will see the duplicate tag warning, which you experienced.
